### ISSUE TITLE
Making elements either more lightweight or lazy

### ISSUE THREAD

**Issue by tinchodias** [pharo-graphics/Bloc#326]
A first analysis, that must be validated in other computers and parameters maybe.
I profiled `(1 to: 1000) collect: [ :each | BlElement new ]` and got:

<img width="381" alt="Screenshot 2023-09-29 at 16 57 13" src="https://github.com/pharo-graphics/Bloc/assets/3044265/f2d1e7d3-8a81-409e-86bf-f19f03ed44aa">

48.8% in new task queues: It is slow because it has a Monitor which has a Semaphore.
The task queue of an element is ONLY used when you add e.g. an animation BEFORE it was added to a space (See `BlElement>>#enqueueTask:`).
Am I wrong to propose creating the element's task queue only if it will be used? I'm not specially expert on this topic, maybe it is a problem if two Pharo processes "in parallel" add the first item to a task queue and then 2 task queues will be created in parallel and only one will prevail at some point, in the element's instvar.

8.7% in new default error handles
I commented about this in #246 too. We have to discuss what behavior we want but I think we can simplify this and reduce this % a lot.

8% in new layout constraints. Here I don't understand since I reports a large time in `BlInsets empty` and this is a singleton:
<img width="367" alt="Screenshot 2023-09-29 at 17 21 51" src="https://github.com/pharo-graphics/Bloc/assets/3044265/73eb60e2-6483-4869-9b9b-4afb5f3facb7">
And `[ BlInsets empty ] bench ">>> 119114148.741 per second"` seems fast.

4.3% in creating a children array.
Many elements will leaf nodes, they don't need to have a collection so early. This has a relation with #321. Maybe we can have a kind of NULL children array that initialized a collection when first child is added.

**Comment by jecisc** [comment]
I tried to profile your piece of code and I get this: 

<img width="738" alt="image" src="https://github.com/pharo-graphics/Bloc/assets/9519971/75aadc6d-f9f1-45da-b2d6-e5b13b512a0d">

Was this improved already?

**Comment by tinchodias** [comment]
I didn't improve it!

**Comment by tinchodias** [comment]
I repeated the profile in this branch: https://github.com/pharo-graphics/Bloc/pull/746 (error handlers creation is optimized), and get:

<img width="535" alt="Image" src="https://github.com/user-attachments/assets/a65339d5-62b1-4441-b8ff-4a35ecffee3f" />
